docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
     volumes:
       - ./api:/src
     ports:
-      - "80:8080"
+      - "8080:8080"
```

Comment:
is this being accounted for on the front end?

Reply:
@Emmanuel-Develops what do you think? i thought it might be better to switch away from the default Apache/Nginx port

Reply:
Either one works for me. I'll simply update on my ends